ui/serializers.py

```diff
@@ -7,6 +7,28 @@
 from rest_framework.settings import api_settings
 
 from ui import models, permissions as ui_permissions
+from ui.utils import get_moira_client
+
+
+def validate_moira_lists(lists):
+    """
+    Raise a validation error if any of the moira lists in a list does not exist or is not a mailing list
+
+    Args:
+        lists(list): List of moira lists
+    """
+    bad_lists = []
+    for mlist in lists:
+        moira_client = get_moira_client()
+        if not moira_client.list_exists(mlist.name):
+            bad_lists.append(mlist.name)
```

Comment:
Can you add a test covering this case?

Reply:
`test_collection_serializer_validation_fake_view_lists`, `test_collection_serializer_validation_fake_admin_lists`

Reply:
It looks like this line is still uncovered: https://codecov.io/gh/mitodl/odl-video-service/pull/316/diff#D4-27

Reply:
Added checks for the contents of the validation exception message which should contain the names of the bad lists.